source/container.puml

```diff
@@ -0,0 +1,33 @@
+@startuml S-ENDA context diagram
+!includeurl https://raw.githubusercontent.com/RicardoNiepel/C4-PlantUML/release/1-0/C4_Container.puml
+
+LAYOUT_LEFT_RIGHT
+
+Person(dataprovider, "Data Provider")
```

Comment:
service producer is missing - is this intentional?

Reply:
No, it's just not finished yet. I tried to give a hint that things might be missing by prefixing the commit with WIP. It's mostly because discarding the pull request didn't work as I expected and it got re-opened when I pushed again to the branch associated with it. Will remember not doing that next time. Now I will just finish it this way and let you know in a message when it's done.